catalog/models.py

```diff
@@ -1,14 +1,17 @@
 import uuid  # Required for unique book instances
 from django.db import models
 from django.urls import reverse
+from django.contrib.auth.models import User
+from datetime import date
+from django.utils.translation import ugettext_lazy as _
```

Comment:
e sắp xếp thứ tự import packages/modules trong file, mục đích để code được sáng, rõ ràng
python packages/modules -> django packages/modules -> user-defined packages/modules

như trong file này thì thứ tự như sau
```suggestion
import uuid  # Required for unique book instances
from datetime import date

from django.db import models
from django.urls import reverse
from django.contrib.auth.models import User
from django.utils.translation import ugettext_lazy as _
```